### PR TITLE
Update for Ports and Expose them

### PR DESCRIPTION
**Pull request by CooliMC**

### CODE REVIEW

File: stretch/Dockerfile [hunk 1]
```diff
@@ -35,11 +35,11 @@ ENTRYPOINT ${STEAMCMDDIR}/steamcmd.sh \
...
-EXPOSE 27166\tcp
-EXPOSE 27166\udp
-EXPOSE 21114\tcp
-EXPOSE 21114\udp
+EXPOSE 7787/udp
+EXPOSE 7788/udp
+EXPOSE 27165/tcp
+EXPOSE 27165/udp
+EXPOSE 27166/tcp
+EXPOSE 27166/udp
```
Comment (CM2Walki) [inline stretch/Dockerfile]: Why do you EXPOSE port 7788 and 27166?

Also as stated before please collapse into one EXPOSE line and use \ to add linebreaks if necessary.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ Running on the *host* interface (recommended):<br/>
 $ docker run -d --net=host --name=squad-dedicated cm2network/squad
 ```
 
+Running as a normal docker container:<br/>
+
+```console
+$ docker run -d -p 7787:7787/udp -p 7788:7788/udp -p 27165:27165/tcp -p 27165:27165/udp -p 27166:27166/tcp -p 27166:27166/udp -p 21114:21114/tcp -p 21114:21114/udp -e PORT=7787 -e QUERYPORT=27165 -e RCONPORT=21115 --name=squad-server cm2network/squad
```
Comment (CM2Walki) [inline README.md]: Same comment as below, what's the reason for you opening port 7788 and 27166? "QueryPort+1" Is only required if you run multiple server instances...

File: stretch/Dockerfile (hunk 1 above)
Comment (CooliMC) [reply stretch/Dockerfile]: I expose all ports like described in the wiki: https://squad.gamepedia.com/Server_Configuration#Ports_to_Open
I also exposed every port by one line for a better overview.

File: README.md (hunk 2 above)
Comment (CooliMC) [reply README.md]: I expose all ports like described in the wiki: https://squad.gamepedia.com/Server_Configuration#Ports_to_Open
Also Port 7788 and 27166 are used if there are problems with the lower default port.

File: stretch/Dockerfile (hunk 1 above)
Comment (CM2Walki) [reply stretch/Dockerfile]: > 
> 
> I expose all ports like described in the wiki: https://squad.gamepedia.com/Server_Configuration#Ports_to_Open

Yes, and QueryPort+1 is for additional game server instances. If you run `lsof` in the container you will never see those ports binded unless specified. This is the same for all Unreal Engine games, they don't have a fallback port mechanic. The Wiki documentation is just off by a bit.

> I also exposed every port by one line for a better overview.

Yes, which is a good thing but there is no need to use a new EXPOSE every new line. This is way better:
```
EXPOSE 7787/udp \
	27165/tcp \
	27165/udp \
	21114/tcp \
	21114/udp
```

File: README.md (hunk 2 above)
Comment (CM2Walki) [reply README.md]: See my reply above

File: stretch/Dockerfile (hunk 1 above)
Comment (CooliMC) [reply stretch/Dockerfile]: Ok changed it

**Pull request merged**